### ISSUE TITLE
State of the project, alternatives?

### ISSUE THREAD

**Issue by guettli** [CTrox/csi-s3-driver#97]
This project seems unmaintained.

Which alternatives do you recommend?

**Comment by zeph** [comment]
not really... there are just one from a russian provider https://github.com/yandex-cloud/k8s-csi-s3 and one from aws (specific to their EKS)
